packages/tools/docker/cluster/docker-compose.yml

```diff
@@ -47,7 +47,7 @@ services:
         
     cluster0:
       container_name: cluster0
-      image: ipfs/ipfs-cluster:latest
+      image: ipfs/ipfs-cluster:v0.14.5-rc1
```

Comment:
@alanshaw I'm going to revert this change to the docker-compose to as the latest is now `v0.14.5` https://github.com/ipfs/ipfs-cluster/releases